pkgs/development/python-modules/fpylll/default.nix

```diff
@@ -46,6 +46,11 @@ buildPythonPackage rec {
     py.test
   '';
 
+  patches = [
+    # https://github.com/fplll/fpylll/issues/121
+    ./readme-encoding.patch
```

Comment:
Upstream patched it 
https://github.com/fplll/fpylll/commit/8ce149bec06d873b07fa650bd507cadf77d99a99
They're fast :)

Reply:
Yeah apparently the author is thinking about cutting a new release that includes that fix, so maybe its worth waiting for that (https://trac.sagemath.org/ticket/25404#comment:8).

Reply:
I've updated to release `0.4.1` which includes the fix. I had to change from `fetchPyPi` to `fetchFromGitHub` since its not on PyPi yet, is that a problem? I think fetching straight from the source is better anyways if pypi doesn't have any other advantages.